discovery-provider/docker-compose.yml

```diff
@@ -32,6 +32,24 @@ services:
     networks:
       - discovery-provider-network
 
+  filebeat:
+    extends:
+      file: ../common-services.yml
+      service: filebeat
+    env_file:
+      - override.env
+    networks:
+      - discovery-provider-network
+
+  metricbeat:
+    extends:
+      file: ../common-services.yml
+      service: metricbeat
+    env_file:
+      - override.env
```

Comment:
Possibly a follow up, but we should not be using override.env to manage these env vars, we should use override only for server specific variables consumed by the protocol repo

Reply:
For sure, feel free to create a follow up ticket and I can handle the reorg you have in mind.

I'm currently following the previously approved method of adding logging containers as seen in #14 .